### ISSUE TITLE
Dark Theme, schedule maintenance, selecting system white/grey on white/grey (almost unreadable)

### ISSUE THREAD

**Issue by BloodyIron** [librenms/librenms#13575]
### The problem

I have the Dark Theme on, and when I schedule maintenance, the colour themeing makes reading the text for selecting servers to add to the list almost impossible, as it's light grey on white background. Can we please have the themeing fixed for this part and other areas where it's still silly colouring? I use Dark Theme everywhere to reduce eyestrain, and I think it's about time this gets properly completed.

### Output of ./validate.php

```txt
$ ./validate.php
====================================
Component | Version
--------- | -------
LibreNMS  | 21.11.0-46-g99fbdeef0
DB Schema | 2021_11_17_105321_device_add_display_field (226)
PHP       | 7.4.3
Python    | 3.8.10
MySQL     | 8.0.27-0ubuntu0.20.04.1
RRDTool   | 1.7.2
SNMP      | 5.8
====================================

[OK]    Composer Version: 2.1.12
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database schema correct
$
```

### What was the last working version of LibreNMS?

Version	21.11.0-46-g99fbdeef0 - Fri Nov 26 2021 10:54:29 GMT-0700

**Comment by murrant** [comment]
Can you please include screenshots?  LibreNMS is a huge application, so if you want someone to fix that for you, please include as much information as you can.

**Comment by smwinn** [comment]
Device summary horizontal and vertical widgets are really bad.
![Screenshot 2021-12-10 115247](https://user-images.githubusercontent.com/58242686/145619295-af3a3891-5110-417a-bd25-25213171dd73.png)

The port screens
![Screenshot 2021-12-10 115603](https://user-images.githubusercontent.com/58242686/145619848-7f2a4684-3c31-4eab-b6d8-c2905c66720f.png)

Really anywhere that blue, dark gray and black is used as a foreground color on the dark background. The red hurts too but not as bad as the blue.

**Comment by BloodyIron** [comment]
https://i.imgur.com/oM8WgLj.png

Look at the "localhost" text. Approaching impossible to read.

**Comment by dfitton** [comment]
Hi All, I have just created a pull request #13749 which corrects some of the issues raised here.
I also found some of the colours in the current dark mode a massive pain to read so a few simple changes have made many things much better.

**Comment by ottorei** [comment]
Fixed by #13749
